server/routes/ivory-integral.route.js

```diff
@@ -1,19 +1,25 @@
+'use strict'
+
+const { Views } = require('../utils/constants')
+
 const handlers = {
   get: (request, h) => {
-    return h.view('ivory-integral', {
+    return h.view(Views.IVORY_INTEGRAL, {
       errorSummaryText: '',
       errorText: false
     })
   },
+
   post: (request, h) => {
     const payload = request.payload
     if (payload.ivoryIsIntegral) {
       const client = request.redis.client
       client.set('ivory-integral', payload.ivoryIsIntegral)
       return h.redirect('check-your-answers')
```

Comment:
This wants to be Views.CHECK_YOUR_ANSWERS

Reply:
Fixed.